Source/WebKit/UIProcess/ios/PageClientImplIOS.mm

```diff
@@ -741,8 +745,16 @@ - (BOOL)_isHostedInAnotherProcess;
     [contentView() _showPlaybackTargetPicker:hasVideo fromRect:elementRect routeSharingPolicy:policy routingContextUID:contextUID.createNSString().get()];
 }
 
-bool PageClientImpl::handleRunOpenPanel(WebPageProxy*, WebFrameProxy*, const FrameInfoData& frameInfo, API::OpenPanelParameters* parameters, WebOpenPanelResultListenerProxy* listener)
+bool PageClientImpl::handleRunOpenPanel(WebPageProxy* proxy, WebFrameProxy*, const FrameInfoData& frameInfo, API::OpenPanelParameters* parameters, WebOpenPanelResultListenerProxy* listener)
```

Comment:
Nit: it might be good to change the proxy parameter from `WebPageProxy*` to `const WebPageProxy&` now that it is being used.

Reply:
Done